src/data-catalog/DataBrowserDetails.ts

```diff
@@ -149,10 +178,10 @@ export const SidebarComponent = ({ dataObj, id }) => {
         isDatarepoSnapshot(dataset),
         async () => {
           setTdrSnapshotPreparePolling(true);
-          const jobInfo = await withErrorReporting(
+          const jobInfo: JobModel = (await withErrorReporting(
```

Comment:
If you have `as JobModel` below then do you need this too? I would expect the inferred type to be `JobModel` given the `as`.